mitodl/ocw-hugo-themes
# ocw-to-hugo 1.19.0

#### Pre-Flight checklist


#### What are the relevant tickets?
None

#### What's this PR do?
This PR is the result of running a `yarn upgrade` to update to the latest version of `ocw-to-hugo`

#### How should this be manually tested?
 - Install dependencies with `yarn install --pure-lockfile`
 - Enter the `ocw-to-hugo` directory with `cd node_modules/@mitodl/ocw-to-hugo`
 - Sync a parsed OCW JSON bucket locally to your computer, such as `open-learning-course-data-rc`
 - Run ocw-to-hugo against the entire folder with a command like node . -i /path/to/open-learning-course-data -o private/output --rm
 - Run `echo $?` and ensure that the exit code is `0`
